src/helpers/submitMsgProposal.ts

```diff
@@ -0,0 +1,31 @@
+import { PersistenceClient } from "../clients/client";
+import { coins } from "@cosmjs/stargate";
+import { Any } from "cosmjs-types/google/protobuf/any";
+
+export async function contractProposal(
+  client: PersistenceClient,
+  proposal: { typeUrl?: string; value?: Uint8Array },
+) {
+  const [account] = await client.wallet.getAccounts();
+  //submit proposal
+  const proposalMsg = {
+    typeUrl: "/cosmos.gov.v1beta1.MsgSubmitProposal",
+    value: {
+      content: Any.fromPartial(proposal),
+      initialDeposit: coins(600_000_000, "stake"),
+      proposer: account.address,
+    },
+  };
+  const res = await client.core.signAndBroadcast(
+    account.address,
+    [proposalMsg],
+    { amount: coins(20_000_000, "stake"), gas: "20000000" },
```

Comment:
some coins are hardcoded to stake/ some to xprt, are these helpers reusable? if not, do you need them to be exported?